google-beta/resource_container_cluster.go

```diff
@@ -2491,6 +2526,21 @@ func flattenPrivateClusterConfig(c *containerBeta.PrivateClusterConfig) []map[st
 	}
 }
 
+func flattenReleaseChannel(c *containerBeta.ReleaseChannel) []map[string]interface{} {
+	result := []map[string]interface{}{}
+	if c != nil {
+		result = append(result, map[string]interface{}{
+			"channel": c.Channel,
+		})
+	} else {
+		// Explicitly set the network policy to the default.
```

Comment:
Not a "network policy", so this comment can be updated.

Why do we want to explicitly set the channel to `UNSPECIFIED` if it is not returned by the API? Typically there is a difference between receiving a value like `UNSPECIFIED` and not receiving anything at all, but we would lose that in this case

Reply:
In this case, when release channel is set to `UNSPECIFIED`, the GKE API returns a `nil` `containerBeta.ReleaseChannel`. As such, I had to make the assumption that the returned `nil` object refers to the `UNSPECIFIED` channel. 

I would add an additional test to test for invalid channels.